app/screens/AddChildSetup.tsx

```diff
@@ -0,0 +1,573 @@
+      handleImportedData(oldChildrenData, importedrealm);
+    }
+  }
+  const importDataIOS = async (): Promise<any> => {
+    DocumentPicker.pick({
+      allowMultiSelection: false,
+      type: DocumentPicker.types.allFiles,
+    })
+      .then(async (res: any) => {
+        console.log(res, "..res..");
+        let oldChildrenData: any = null;
+        let importedrealm: any = null;
+        if (res.length > 0 && res[0].uri) {
+          if (res[0].name.endsWith(".json")) {
+            const exportedFileContent: any = await RNFS.readFile(decodeURIComponent(res[0].uri), 'utf8');
+            const decryptedData = decryptData(exportedFileContent, encryptionsKey)
+              .then((text: any) => {
+                return text;
+              })
+              .catch((error: any) => {
+                console.log("Decrypted error", error);
+                throw error;
+              });
+            const importedData = JSON.parse(await decryptedData);
+            await RNFS.writeFile(tempRealmFile, JSON.stringify(decryptedData), "utf8");
+            oldChildrenData = importedData;
+          } else {
+            const exportedFileContent: any = await RNFS.readFile(decodeURIComponent(res[0].uri), 'base64');
+            await RNFS.writeFile(tempRealmFile, exportedFileContent, "base64");
+            importedrealm = await new Realm({ path: 'user1.realm' });
+            if (importedrealm) {
+              importedrealm.close();
+            }
+            importedrealm = await new Realm({ path: 'user1.realm' });
+            const user1Path = importedrealm.path;
+            console.log(user1Path, "..user1Path");
+            oldChildrenData = importedrealm.objects('ChildEntity');
+          }
+
+          setImportAlertVisible(false);
+          setLoading(true);
+          setIsImportRunning(true);
+          handleImportedData(oldChildrenData, importedrealm)
+
+        }
+
+      })
+      .catch(handleError);
+  }
+  const importFromFile = async (): Promise<any> => {
+    if (Platform.OS == "android") {
+      importDataAndroid();
+    } else {
+      importDataIOS();
+    }
+  }
+  const importAllData = async (): Promise<any> => {
+    setImportAlertVisible(false);
+    setLoading(true);
+    setIsImportRunning(true);
+    //param 1 from settings import for navigation
+    const importResponse = await backup.import1(navigation, languageCode, dispatch, childAge, genders);
+    if (importResponse.length > 0) {
+      setIsImportRunning(false);
+      setLoading(false);
+      navigation.navigate('ChildImportSetup', {
+        importResponse: JSON.stringify(importResponse)
+      });
+    }
+    else {
+      setLoading(false);
+      setIsImportRunning(false);
+    }
+  }
+
+
+  const AddChild = async (isDefaultChild: boolean,isDefaultName: boolean): Promise<any> => {
+    await userRealmCommon.getData<ChildEntity>(ChildEntitySchema);
+    let defaultName ;
+    if(isDefaultName){
+       defaultName='Baby';

```

Comment:
Use constant to store string 'Baby'

Reply:
It's already we use in development branch. when we merge the code it will replace.